### ISSUE TITLE
Non fatal: Null check operator used on a null value. EditableTextState.toggleToolbar [editable_text.dart:3123])

### ISSUE THREAD

**Issue by rohansohonee1** [flutter/flutter#120979]
## Steps to Reproduce
Unable to reproduce. Crash reported from firebase crashlytics.

**Expected results:** 
Should not crash.

**Stack trace:** 

```dart
Non-fatal Exception: io.flutter.plugins.firebase.crashlytics.FlutterError: Null check operator used on a null value. Error thrown Instance of 'Ria'.
#00 pc 0x8e59f7 com.agendaboa.app (EditableTextState.toggleToolbar [editable_text.dart:3123])
#01 pc 0x8e769b com.agendaboa.app (_TextFieldState._handleSelectionHandleTapped [text_field.dart:1081])
#02 pc 0x8e761b com.agendaboa.app (_TextFieldState._handleSelectionHandleTapped [text_field.dart:1079])
#03 pc 0xcfd78f com.agendaboa.app (GestureRecognizer.invokeCallback [recognizer.dart:253])
#04 pc 0xb948f7 com.agendaboa.app (TapGestureRecognizer.handleTapUp [tap.dart:627])
#05 pc 0x4c02b7 com.agendaboa.app (BaseTapGestureRecognizer._checkUp [tap.dart:306])
#06 pc 0x4c01fb com.agendaboa.app (BaseTapGestureRecognizer.acceptGesture [tap.dart:276])
#07 pc 0x4be49f com.agendaboa.app (GestureArenaManager.sweep [arena.dart:163])
#08 pc 0x5048e3 com.agendaboa.app (GestureBinding.handleEvent [binding.dart:464])
#09 pc 0xce9ec3 com.agendaboa.app (GestureBinding.dispatchEvent [binding.dart:440])
#10 pc 0x3c69ef com.agendaboa.app (RendererBinding.dispatchEvent [binding.dart:337])
#11 pc 0x3c694b com.agendaboa.app (GestureBinding._handlePointerEventImmediately [binding.dart:395])
#12 pc 0x3c62f3 com.agendaboa.app (GestureBinding._flushPointerEventQueue [binding.dart:357])
#13 pc 0x604583 com.agendaboa.app (GestureBinding._handlePointerDataPacket [binding.dart:295])
#14 pc 0x60448f com.agendaboa.app (GestureBinding._handlePointerDataPacket [binding.dart:290])
#15 pc 0x399e0f com.agendaboa.app (_invoke1 [hooks.dart:167])
#16 pc 0x399d57 com.agendaboa.app (PlatformDispatcher._dispatchPointerDataPacket [platform_dispatcher.dart:341])
#17 pc 0x399ce3 com.agendaboa.app (_dispatchPointerDataPacket [hooks.dart:94])
```

**Flutter version**
Stable channel 3.3.8

**Device details:**
Brand:samsung
Model:Galaxy J2 Pro
Orientation: Portrait
RAM free: 283.8 MB
Disk free: 576.64 MB
Operating System
Version:Android 7.1.1
Orientation: Portrait
Rooted:No
Crash
Date:Feb 16, 2023, 3:27:49 PM

**Comment by gnprice** [comment]
This appears to be the same issue as #64471. It was fixed in Flutter 3.7.

**Comment by exaby73** [comment]
Hello @rohansohonee1. I suggest you upgrade you Flutter version to 3.7 :) But indeed, this looks like a duplicate of https://github.com/flutter/flutter/issues/64471. Closing this issue for the same reason. Thank you
